docs/api.md

```diff
@@ -1585,7 +1585,7 @@ Shortcut for [page.mainFrame().title()](#frametitle).
   - `timeout` <[number]> Maximum time in milliseconds, defaults to 30 seconds, pass `0` to disable timeout. The default value can be changed by using the [browserContext.setDefaultTimeout(timeout)](#browsercontextsetdefaulttimeouttimeout) or [page.setDefaultTimeout(timeout)](#pagesetdefaulttimeouttimeout) methods.
 - returns: <[Promise]>
 
-Sends a `keydown`, `keypress`/`input`, and `keyup` event for each character in the text.
+Sends a `keydown`, `keypress`/`input`, and `keyup` event for each character in the text. `page.type` can be used to send fine-grained keyboard events. To fill text in form fields, use [`page.fill`](#pagefillselector-value-options).
```

Comment:
- `fill text` -> `fill values` ?
- maybe add an opposite statement to the fill method?

Reply:
Thanks! Made both the changes